src/GrommetSwagger.js

```diff
@@ -70,10 +70,11 @@ export default class GrommetSwagger extends Component {
   }
 
   render() {
-    const { background, executable } = this.props;
+    const { background, executable, data: propsData } = this.props;
     const {
-      contextSearch, data, error, history, loading, origin, theme, url,
+      contextSearch, data: stateData, error, history, loading, origin, theme, url,
     } = this.state;
+    const data = { ...stateData, ...propsData };
```

Comment:
What's the rationale for having the props version override the state version? I would have expected them to be ordered the other way.

Reply:
It's giving the users some control over the data because the configuration object is being passed in to `Swagger.mount`, which I thought it is the configuration.

Reply:
state version has some wrong data, for example, which is not controllable by users, then users can fix them easily from ui side.

Reply:
If the data in the swagger is wrong, why isn't the correct solution to fix the swagger?

Reply:
alright, I made the edit. Thanks!